test/addons/rook-cephfs/snapshot-class.yaml

```diff
@@ -13,7 +13,7 @@ kind: VolumeSnapshotClass
 metadata:
   name: csi-cephfsplugin-snapclass
   labels:
-    ramendr.openshift.io/storageid: rook-cephfs-$cluster-1
+    ramendr.openshift.io/storageid: rook-cephfs-$fsname-$cluster-1
```

Comment:
why we don't use the storage class name instead of hardcoding "rook-cephfs"?

Reply:
updated.